### PR TITLE
readme and changelog boilerplate text

### PR DESCRIPTION
**Pull request by hemanttanwar**
Before the release we need to make sure that all of the docstrings are finalized and updated before the package is released as this is the version that will be publicly available in documentation. Refer to teams for more detailed guidance
 
1. READMEs per package done and tested
 
2.  Docstrings per package updated

https://github.com/Azure/azure-sdk-for-java/issues/5935

### CODE REVIEW

File: sdk/storage/azure-storage-file/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Shares provide a way to organize sets of files and also can be mounted as an SMB
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-file</artifactId>
-  <version>12.0.0-preview.5</version>
+  <version>12.0.0</version>
```
Comment (joshfree) [inline sdk/storage/azure-storage-file/README.md]: file should be 12.0.0-preview.5

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # Release History
+## 12.0.0 (2019-10-31)
```
Comment (joshfree) [inline sdk/identity/azure-identity/CHANGELOG.md]: 2019-10-29

File: sdk/storage/azure-storage-file/README.md (hunk 1 above)
Comment (joshfree) [reply sdk/storage/azure-storage-file/README.md]: @sima-zhu

File: sdk/identity/azure-identity/CHANGELOG.md (hunk 2 above)
Comment (hemanttanwar) [reply sdk/identity/azure-identity/CHANGELOG.md]: Is this change only for identity CHANGELOG or for every other CHANGELOG ?

File: sdk/eventhubs/azure-messaging-eventhubs/README.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ documentation][event_hubs_product_docs] | [Samples][sample_examples]
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-messaging-eventhubs</artifactId>
-    <version>5.0.0-preview.5</version>
```
Comment (samvaity) [inline sdk/eventhubs/azure-messaging-eventhubs/README.md]: I don't think we are GA'ing eventhubs in the November release.

**Comment by g2vinay** [comment]
Below are the changelogs for KV GA libraries.

Changelog for KV Keys: 
## 4.0.0-preview.5

### Breaking changes

- `Key` has been renamed to `KeyVaultKey` to avoid ambiquity with other libraries and to yield better search results.
- `Key.keyMaterial` has been renamed to `KeyVaultKey.key`.
- The setters of `JsonWebKey` properties have been removed.
- `JsonWebKey` methods `fromRsa`, `fromEc` and `fromAes` now take an optional collection of key operations.
- `JsonWebKey.keyOps` is now read-only. You must pass a list of key operations at construction time.
- `endpoint` method on `KeyClientBuilder` has been renamed to `vaultEndpoint`.
- `hsm` properties and parameters have been renamed to `hardwareProtected`.
- On `KeyProperties`, `expires`, `created`, and `updated` have been renamed to `expiresOn`, `createdOn`, and `updatedOn` respectively.
- On `DeletedKey`, `deletedDate` has been renamed to `DeletedOn`.
- `KeyClient.listKeys` and `KeyClient.listKeyVersions` have been renamed to `KeyClient.listPropertiesOfKeys` and `KeyClient.listPropertiesOfKeyVersions` respectively.
- `KeyClient.restoreKey` has been renamed to `KeyClient.restoreKeyBackup` to better associate it with `KeyClient.backupKey`.
- `KeyClient.deleteKey` has been renamed to `KeyClient.beginDeleteKey` and now returns a `Poller` to track this long-running operation.
- `KeyClient.recoverDeletedKey` has been renamed to `KeyClient.beginRecoverDeletedKey` and now returns a `Poller` to track this long-running operation.
- `KeyCreateOptions` has been renamed to `CreateKeyOptions`.
- `EcCreateKeyOptions` has been renamed to `CreateEcKeyOptions`.
- `CreateEcKeyOptions.curve` has been renamed to `curveName` to be consistent.
- `RsaKeyCreateOptions` has been renamed to `CreateRsaKeyOptions`.
- `KeyImportOptions` has been renamed to `ImportKeyOptions`.

### Major changes

- `JsonWebKey.keyType` and `JsonWebKey.keyOps` have been exposed as `KeyVaultKey.keyType` and `KeyVaultKey.keyOperations` respectively.
- `KeyClient.vaultEndpoint` has been added with the original value pass to `KeyClient`.



Changelog for Kv Secrets: 
### Breaking changes

- `Secret` has been renamed to `KeyVaultSecret` to avoid ambiquity with other libraries and to yield better search results.
- `endpoint` method on `SecretClientBuilder` has been renamed to `vaultEndpoint`.
- On `SecretProperties`, `expires`, `created`, and `updated` have been renamed to `expiresOn`, `createdOn`, and `updatedOn` respectively.
- On `DeletedSecret`, `deletedDate` has been renamed to `deletedOn`.
- `SecretClient.listSecrets` and `SecretClient.listSecretVersions` have been renamed to `SecretClient.listPropertiesOfSecrets` and `SecretClient.listPropertiesOfSecretVersions` respectively.
- `SecretClient.restoreSecret` has been renamed to `SecretClient.restoreSecretBackup` to better associate it with `SecretClient.backupSecret`.
- `SecretClient.deleteSecret` has been renamed to `SecretClient.beginDeleteSecret` and now returns a `Poller` to track this long-running operation.
- `SecretClient.recoverDeletedSecret` has been renamed to `SecretClient.beginRecoverDeletedSecret` and now returns a `Poller` to track this long-running operation.

### Major changes

- `SecretClient.vaultEndpoint` has been added with the original value pass to `SecretClient`.

File: sdk/storage/azure-storage-file/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## Version 12.0.0 (2019-10-31)
```
Comment (sima-zhu) [inline sdk/storage/azure-storage-file/CHANGELOG.md]: File will not go with GA but another round of preview. Might not need to change here.

File: sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,18 @@
 # Change Log azure-data-appconfiguration
 
+## Version 1.0.0 (2019-10-25)
+For details on the Azure SDK for Java (October 2019 Preview) release refer to the [release announcement](https://aka.ms/azure-sdk-preview6-java).
+- Ensure exceptions are consistent for certain operations (c.f. other languages)
```
Comment (samvaity) [inline sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md]: Should be `https://aka.ms/azure-sdk-**preview6**-java`, `preview5` instead?

File: sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md [hunk 6]
```diff
@@ -1,4 +1,29 @@
...
+- KeyClient.recoverDeletedKey has been renamed to KeyClient.beginRecoverDeletedKey and now returns a Poller to track this long-running operation.
+- KeyCreateOptions has been renamed to CreateKeyOptions.
+- EcCreateKeyOptions has been renamed to CreateEcKeyOptions.
+- CreateEcKeyOptions.curve has been renamed to curveName to be consistent.
+- RsaKeyCreateOptions has been renamed to CreateRsaKeyOptions.
+- KeyImportOptions has been renamed to ImportKeyOptions.
+
+### Major changes
+- JsonWebKey.keyType and JsonWebKey.keyOps have been exposed as KeyVaultKey.keyType and KeyVaultKey.keyOperations respectively.
+- KeyClient.vaultEndpoint has been added with the original value pass to KeyClient.
```
Comment (g2vinay) [inline sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md]: replace vaultEndpoint with vaultUrl

File: sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md [hunk 7]
```diff
@@ -1,4 +1,29 @@
 # Release History
+## 4.0.0 (2019-10-31)
+
+### Breaking changes
+- Key has been renamed to KeyVaultKey to avoid ambiquity with other libraries and to yield better search results.
+- Key.keyMaterial has been renamed to KeyVaultKey.key.
+- The setters of JsonWebKey properties have been removed.
+- JsonWebKey methods fromRsa, fromEc and fromAes now take an optional collection of key operations.
+- JsonWebKey.keyOps is now read-only. You must pass a list of key operations at construction time.
+- endpoint method on KeyClientBuilder has been renamed to vaultEndpoint.
```
Comment (g2vinay) [inline sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md]: replace vaultEndpoint with vaultUrl

File: sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md [hunk 8]
```diff
@@ -1,4 +1,19 @@
 # Release History
+## 4.0.0 (2019-10-31)
+### Breaking changes
+
+- Secret has been renamed to KeyVaultSecret to avoid ambiquity with other libraries and to yield better search results.
+- endpoint method on SecretClientBuilder has been renamed to vaultEndpoint.
```
Comment (g2vinay) [inline sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md]: replace vaultEndpoint with vaultUrl

File: sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md [hunk 9]
```diff
@@ -1,4 +1,19 @@
...
+- On SecretProperties, expires, created, and updated have been renamed to expiresOn, createdOn, and updatedOn respectively.
+- On DeletedSecret, deletedDate has been renamed to deletedOn.
+- SecretClient.listSecrets and SecretClient.listSecretVersions have been renamed to SecretClient.listPropertiesOfSecrets and SecretClient.listPropertiesOfSecretVersions respectively.
+- SecretClient.restoreSecret has been renamed to SecretClient.restoreSecretBackup to better associate it with SecretClient.backupSecret.
+- SecretClient.deleteSecret has been renamed to SecretClient.beginDeleteSecret and now returns a Poller to track this long-running operation.
+- SecretClient.recoverDeletedSecret has been renamed to SecretClient.beginRecoverDeletedSecret and now returns a Poller to track this long-running operation.
+
+### Major changes
+- SecretClient.vaultEndpoint has been added with the original value pass to SecretClient.
```
Comment (g2vinay) [inline sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md]: replace vaultEndpoint with vaultUrl

File: sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md (hunk 7 above)
Comment (g2vinay) [reply sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md]: This change just went in today.

File: sdk/core/azure-core-tracing-opencensus/CHANGELOG.md [hunk 10]
```diff
@@ -1,13 +1,19 @@
...
+- Fixed Service loader provider configuration file to correctly trace HTTP requests.
+- Moved package under azure-core directory.
+- Removed client libraries dependencies from the package.
+- Moved ProcessKind enum to public tracing folder under azure-core.
+- Updated samples to be markdown files.
 See this package's
   [documentation](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core-tracing-opencensus/README.md) and
   [samples](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/core/azure-core-tracing-opencensus/src/samples) demonstrate the new API.
 
+
```
Comment (samvaity) [inline sdk/core/azure-core-tracing-opencensus/CHANGELOG.md]: nit: extra line

**Pull request merged**